backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs

```diff
@@ -140,4 +141,15 @@ public async Task<List<TeamDto>> GetCurrentUserTeams()
 
         return _mapper.Map<List<TeamDto>>(teams);
     }
+    
+    public async Task<List<TeamDto>> GetCurrentUserAdminTeams()
+    {
+        var currentUser = await _userService.GetCurrentUserAsync();
```

Comment:
Why not just the current user Id?

Reply:
That method was private and I didn't noticed it. However, I can fix it

Reply:
In addition, existing method return uid, but long id is required

Reply:
I can create new method which will return id

Reply:
We have private service method which returns uid, but long id is required and can be accessed only after request to db

Reply:
We will need to rewrite this after merging https://github.com/BinaryStudioAcademy/bsa-2022-easymeets/pull/148